src/ResourceManagement/ServerManagement/ServerManagement/Microsoft.Azure.Management.ServerManagement.nuget.proj

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <!--
+    Microsoft.Azure.Management.ServerManagement
+    -->
+    <SdkNuGetPackage Include="Microsoft.Azure.Management.ServerManagement">
+      <PackageVersion>1.0.3</PackageVersion>
```

Comment:
Just for clarity any suffix to the version number starting with a '-' signals to nuget that the library is parerlease, if someone selects the 'Stable Only' selector, it will not show up.  This is generally what SDKs do for their first version.

Also, since we like to use semantic versioning, starting with 1 menas you're going to update the major version for every breaking change.  It is safer to start with 0. unless the library is fairly stable, because semver rules don't apply then.